### DISCUSSION TITLE
GPS on iGate

### DISCUSSION THREAD

**Discussion by OlivierTE** [lora-aprs/LoRa_APRS_iGate#149] [Ideas]
Hello,
I am happy to use your code to make my two LORA modules work.
It's fantastic to see them appear on the APRS.FI map.
In order to adapt this tool to civil security, of which I am a part, do you think it is possible for the iGATE to be able to modify its GPS positioning on its own, as if it were a tracker?
I manage to connect the iGate to the internet via the connection sharing of my smartphone but it keeps its position described in the programming code.
Thank you for your help and congratulations again for your work.
73 - F4CEO - Olivier

**Comment by peterus** [comment]
Hi Oliver!
Can you explain more your case? In ham radio all the iGates are 99% static from the position. So you found one exception ;)
Is this for SAR? And you want to place an iGate close to the operational command and give the groups a tracker?
As I am working also for the austrian red cross, I am happy to here what you want to do :)

**Comment by OlivierTE** [comment]
Hello and thank you for responding so quickly.
You understood my need.
1 iGate in the command post and trackers for the radio amateurs doing
the research.
I have 2 MESTASTIC modules which I briefly tested with their software
installed. From what I have seen and read on the internet, they
communicate with each other because they are both iGate and tracker.
This logic could prove to be more effective than that of the standard
APRS because even if the command post does not hear a module, the
others can relay the positioning and thus obtain the information on
APRS.FI.
is it technically feasible?
Thank you
F4CEO


Le dim. 23 janv. 2022 à 21:58, Peter Buchegger ***@***.***> a
écrit :

**Comment by vk4ypb** [comment]
I agree that this would be a good idea. Even though you think 99% of igates are static. For WICEN activities for instance this wold be valuable as the igate will be in different locations for different events. And why not. You could still use the hard-coded location as the starting point, then use the GPS signal location once a lock was established. These devices are portable, so to not allow the gates location to also GPS signal is not logical.
Thanks.
Peter
VK4YPB

**Reply by peterus** [reply]
Hi Peter!
In the options you can find a gps configuration option which can be activated. Can you check with an T-Beam (has build in GPS) if this is something you are looking for?
Thanks!
